interface/components/export-code/exporter.js

```diff
@@ -224,6 +224,53 @@ module.exports = {
         }
         output += ");\n}";
 
+        return output;
+    },
+
+
+
+    rFunc(){
```

Comment:
Please move this code to another branch. It does not belong wiht graph download button.